<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="prefix_lbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>32</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="addPref_apply_btn_2">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="undo_btn_1">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QCheckBox" name="Hi_checkBx_2">
      <property name="text">
       <string>Hierarchy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="Bfr_checkBx_2">
      <property name="text">
       <string>Before</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="pref_line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="add_pref_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>164</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD any prefix to a selection:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
